src/rbus/rbus_subscriptions.c

```diff
@@ -787,7 +811,8 @@ void rbusSubscriptions_resubscribeRowElementCache(rbusHandle_t handle, rbusSubsc
                 {
                     RBUSLOG_INFO("resubscribing %s for %s", sub->eventName, sub->listener);
                     rtList_RemoveItem(subscriptions->subList, item, NULL);
-                    err = subscribeHandlerImpl(handle, true, el, sub->eventName, sub->listener, sub->componentId, sub->interval, sub->duration, sub->filter);
+                    subscriptionId = &sub->subscriptionId;
```

Comment:
We need to revisit the logic of Provider crash n recovery stuff.
Because, the very first row may be subscribed very later & the subscriptionID is 1.
But when crash & recovered, this method will be called while registering a row and it will get subscriptionID as 25.

The consumer thinks that the subscriptionID is 1 but the provider is having it as 25.